amigo_api.py

```diff
@@ -28,13 +29,29 @@ def fetch_project_list(self):
             return []
 
     def fetch_dataset_list(self, project_id):
-        dataset_url = self.url + '/api/v1/users/0/projects/' + project_id + '/datasets?summary'
+        dataset_url = self.url + '/api/v1/users/0/projects/' + project_id + '/datasets'
```

Comment:
Why are we switching away from ?summary? There are implications of this, it makes calls slower, also the page size becomes 20 instead of 100..